### DISCUSSION TITLE
Export CSV in multiple file

### DISCUSSION THREAD

**Discussion by syji35** [nocodb/nocodb#2536] [Q&A]
I have a table with 14K records. 

The CSV export generates multiple files. Is it possible to configure the export to have only 1 file ? 

NocoDB used as docker : true 
NocoDB version : 0.91.7
Database used in NC_DB URL  : postgresql 
Project was created  by clicking : Export Airtable
Database on which spreadsheet is created :  pg 
Database version :  14

**Steps To Reproduce**

1. Go to Your project
2. Click on 'Download as csv'
3. browser download multiple file

**Comment by pranavxc** [comment]
You can set `NC_EXPORT_MAX_TIMEOUT` environment value default is `5000` (in milliseconds). It will take as much data as possible within that time so you can set it with a higher value to get more data in a single request.

**Reply by syji35** [reply]
Thanks for the answer.

I think it would be interesting to have an asynchronous export, in order to export a big table for non-IT users.
